.github/workflows/release-announcments.yml

```diff
@@ -0,0 +1,68 @@
+name: release-announcements
```

Comment:
Uh, why is this CI file duplicated? Please check the `tools` template for the correct one... this one doesn't look good (has typo in the name)

Reply:
![image](https://github.com/user-attachments/assets/d92ce295-db8f-4c44-9e7a-70b25137cd30)

Indeed, this file should be deleted